src/components/projects-summary/projects-summary.html

```diff
@@ -99,4 +99,25 @@ <h2 class="secondary" ng-if="$ctrl.projects && $ctrl.projects.length > 0">Gettin
       </ul>
     </div>
   </div>
+  <div ng-if="$ctrl.recentlyViewedItems.length">
+    <h2 class="secondary">Recently Viewed</h2>
+    <div class="services-view">
+      <div class="services-view-container">
+        <div class="services-items-inner">
+          <div class="services-items-inner-shadow-covers">
+            <a href="" class="services-item" ng-repeat="item in $ctrl.recentlyViewedItems track by (item.resource | uid)"
```

Comment:
hmmm ... regardless if it's alphabetical or not, you will still see things move :) won't you ? Maybe less frequently, but it'll still happen.  So i'm not sure that alphabetizing it "fixes" the issue.  @beanh66 thoughts?